pkgs/applications/networking/instant-messengers/zoom-us/default.nix

```diff
@@ -24,11 +30,17 @@ in stdenv.mkDerivation {
   buildInputs = [
     dbus glib libGL libX11 libXfixes libuuid libxcb qtbase qtdeclarative
     qtlocation qtquickcontrols2 qtscript qtwebchannel qtwebengine
+    libjpeg_turbo pciutils procps
```

Comment:
I think `pciutils` and `procps` shouldn't be mentioned here. We specify them as build inputs as arguments in package args, but those shouldn't be discovered later by build script - we refer to those explicitly.

IMO `buildInputs` are formed from packages that are used implicitly by build script. pciutils and procps are noop here.